pkg/credentials/revocation_status/utils.go

```diff
@@ -15,12 +14,8 @@ func buildRHSRevocationURL(host, issuerState string) string {
 	return fmt.Sprintf("%s/node?state=%s", host, issuerState)
 }
 
-func buildDirectRevocationURL(host, issuerDID string, nonce uint64, singleIssuer bool) string {
-	if singleIssuer {
-		return fmt.Sprintf("%s/v1/credentials/revocation/status/%d",
-			host, nonce)
-	}
-	return fmt.Sprintf("%s/v1/%s/claims/revocation/status/%d", host, url.QueryEscape(issuerDID), nonce)
+func buildDirectRevocationURL(host string) string {
```

Comment:
I would rename buildDirectRevocationURL to build agent url, not confuse developers in other places.

Reply:
I will remove the function because it is no longer needed